### PR TITLE
Redux conversion

### PR DESCRIPTION
**Pull request by ggarcia852**
hey bro this my initial conversion to redux. still have some converting to do but wanted to get this submitted for now

### CODE REVIEW

File: src/store/coinPageData/index.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+const initialState = {
+  isLoading: false,
+  hasError: false,
+    marketData: null,
+    coinData: null,
+    chartData: null,
+    chartDays: 30,
+  };
```
Comment (yoloOnTheBattlefield) [inline src/store/coinPageData/index.js]: some weird indent in here

**Review by yoloOnTheBattlefield — APPROVED** [review]

File: src/components/ConversionBar/ConversionBar.js [hunk 2]
```diff
@@ -7,25 +8,39 @@ import {
...
 
   const handleCurrencyChange = (e) => {
     setConversionAmount(e.target.value);
-    setCoinAmount(((1 / props.coinPrice) * e.target.value).toFixed(6));
+    setCoinAmount(
+      (1 / props.coinPrice) * e.target.value > 0.1 ||
+        (1 / props.coinPrice) * e.target.value === 0
+        ? ((1 / props.coinPrice) * e.target.value).toFixed(2)
+        : ((1 / props.coinPrice) * e.target.value).toFixed(5)
+    );
```
Comment (yoloOnTheBattlefield) [inline src/components/ConversionBar/ConversionBar.js:33]: jesus, what's this? what are you trying to accomplish?

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/ConversionBar/ConversionBar.js (hunk 2 above)
Comment (ggarcia852) [reply src/components/ConversionBar/ConversionBar.js:33]: its for the number of decimals places showing depending how big the number is. the best I could come up with at the time. wasnt working when I tried importing a formula from utils

Comment (ggarcia852) [reply src/components/ConversionBar/ConversionBar.js:33]: but if thats the only thing you are questioning im ok with it :)

**Pull request merged**